### ISSUE TITLE
Implement UPnP feature: GetFeatureList

### ISSUE THREAD

**Issue by SubJunk** [UniversalMediaServer/UniversalMediaServer#2597]
It's a "required" endpoint that we don't respond to

http://upnp.org/specs/av/UPnP-av-ConnectionManager-v3-Service.pdf

**Comment by SubJunk** [comment]
@ik666 I wonder if you have any interest in doing this to improve our UPnP support, since you have commented on UPnP things a lot

**Comment by ik666** [comment]
@SubJunk I'll have a look. You don't want to implement clock synchronization, but just the response, that the media server device doesn't support it, right?

**Comment by SubJunk** [comment]
Yeah we could have a different issue for that if we want it, it makes sense to build it iteratively I think
